core/src/main/javascript/app/components/JobSelector.js

```diff
@@ -292,7 +292,6 @@ const styles = {
       }
     },
     "& .Select-value": {
-      lineHeight: "1em",
```

Comment:
Are you sure of that? Was it a warning from eslint? Because it could override another rule.